app/assets/stylesheets/components/_email-signup-section.scss

```diff
@@ -0,0 +1,38 @@
+.app-c-email-signup {
+  @include govuk-responsive-margin(8, "top");
+  padding: govuk-spacing(6);
+  background-color: govuk-colour("light-grey", $legacy: "grey-4");
+  border: 1px solid $govuk-border-colour;
+}
+
+.app-c-email-signup--full-width {
+  margin-top: 0;
+  border-left: 0;
+  border-right: 0;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.app-c-email-signup--transparent_background {
+  background: none;
+  border: none;
+}
+
+.app-c-email-signup--no-intro .govuk-link {
+  @include govuk-font(19, $weight: bold);
+  display: block;
```

Comment:
With this as `block` the focus state is very large, suggest using `inline-block` instead (will need to adjust the margin left as well)

<img width="898" alt="Screenshot 2020-05-26 at 11 39 02" src="https://user-images.githubusercontent.com/861310/82891459-af402d80-9f45-11ea-9b66-1d6dab483d8b.png">